app/assets/stylesheets/splash.scss

```diff
@@ -6,9 +11,130 @@
   -webkit-background-size: cover;
   background-size: cover;
   height: 50vh;
-  position: relative;
+  overflow: hidden;
+  box-shadow: 0 0 5px rgba(0,0,0,1);
 }
 
 .bottom {
   height: 50vh;
+  color: $black;
+}
+
+// *************************** TITLE BOX **************************************
+
+.title-box {
+  top: 20%;
+  height: 50%;
+  text-align: center;
+  position: relative;
+  width: 100%;
+}
+
+.title {
+  color: $white;
+  font-family: 'Josefin Sans', sans-serif;
+  font-size: 10vmin;
+}
+
+.subtitle {
+  font-size: 7vmin;
+}
+
+.action {
+  margin-top: 5vh;
```

Comment:
Properties should be ordered background, border, font-family, font-size, margin-top